styles/main.css

```diff
@@ -255,34 +261,48 @@ header nav .burger {
         display: block;
     }
     header nav ul {
-        display: block;
-        direction: ltr;

```

Comment:
This is not how to make a flex element use a vertical layout.  See the flex-direction declaration below